src/twisted/topfiles/8069.bugfix

```diff
@@ -0,0 +1 @@
+The transition to the hyperlink package adds IPv6 support to twisted.python.url.URL.
```

Comment:
I don't see the point of advertising something about a deprecated functionality.